docroot/profiles/custom/cgov_site/modules/custom/cgov_infographic/cgov_infographic.module

```diff
@@ -78,12 +78,15 @@ function cgov_infographic_media_view(array &$build,
 
   // If we're rendering an infographic, populate / override the mobile
   // alt with the desktop version. (Note:field_infograpic is a required field.)
-  $infographic_desktop_alt = $entity->get('field_infographic')->alt;
+   /** @var \Drupal\media\Entity\Media $entity */
+  $infographic_desktop_alt = $entity->get('field_infographic')->getValue()[0]['alt'];
 
   if (!empty($infographic_desktop_alt)) {
     // Set the mobile alt tag for this render to the desktop version.
+     /** @var \Drupal\media\Entity\Media $entity */
     $mobile = $entity->get('field_infographic_mobile');
     if ($mobile) {
+      /* @phpstan-ignore-next-line */
```

Comment:
Please fix this to not ignore.

Reply:
Access to an undefined property alt, not sure what to update here.